### ISSUE TITLE
Les données du stream OSC s'interromptent de manière imprévisible

### ISSUE THREAD

**Issue by sofian** [a3lab/morphoses#91]
C'est un problème critique. Je suppose que c'est dû à une forme de surcharge du réseau.

Afin d'éviter le problème lié à la réception des données qui s'interrompt, remplacer l'envoi des données OSC par des messages MQTT.

Essayer la librairie [PicoMQTT](https://www.hackster.io/news/michal-lesniewski-s-picomqtt-offers-performant-mqtt-client-broker-operation-on-esp32-esp8266-mcus-7a55ed1ba102)

**Comment by sofian** [comment]
Bug critique priorité no 1.

**Comment by sofian** [comment]
@etiennemontenegro @maxdee Discussion about IMU BNO080 and the library we are using and i2c hang: https://github.com/sparkfun/SparkFun_BNO080_Arduino_Library/issues/72

**Comment by sofian** [comment]
We ran all the three robots with my Python script (multithreaded). After about 10 minutes, my laptop stops receiving the UDP packets from the ESP32s. I looked with oscdump *and* with Wireshark: no UDP packets. However, the messages are still being sent: they are broadcasted and we are able to receive them on other laptops! I had to restart the network manager to start getting packets again.

So in other words it looks like my Python script somehow "broke" my capacity to receive UDP messages.

I am suspicious about the multithreading / OSC interactions as the culprit. It's not that the port keeps being used by the process (I think) because when this is the case, normally, oscdump just crashes and emits an error message. Here, it runs but I receive nothing. I also receive no UDP messages according to wireshark.

So the cause (or at least one of the causes) seems to be in the OSC Python libraries I am using on my Linux box. There is something really wrong happening there.
